operations/mimir-mixin/alerts/compactor.libsonnet

```diff
@@ -98,8 +98,8 @@
           alert: $.alertName('CompactorSkippedBlocksWithOutOfOrderChunks'),
           'for': '1m',
           expr: |||
-            increase(cortex_compactor_blocks_marked_for_no_compaction_total{component="compactor", reason="block-index-out-of-order-chunk"}[5m]) > 0
```

Comment:
Why is `component` an issue? That's unrelated from the deployment mode.

Reply:
Follow up after a quick chat on Slack:

- Josh: there is no  shows there is no component label on `cortex_compactor_blocks_marked_for_no_compaction_total`. So I actually think the alert is broken for all deployment modes.
- Marco: You're right it's broken. Anyway, you don't need any selector (e.g. no "job"). The key point is that `cortex_compactor_blocks_marked_for_no_compaction_total` is a metric that can only be exposed by compactor, so you don't need to disambiguate it. Just removing `component="compactor"` is fine.